packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -78,6 +78,7 @@ const BlockDraggableWrapper = ( { children } ) => {
 		width: useSharedValue( 0 ),
 		height: useSharedValue( 0 ),
 	};
+	const currentYPosition = useSharedValue( 0 );
```

Comment:
I decided to create a new variable for this, I guess we could use the `chip` data but it'd be a bit confusing to be passing a variable with that name to the `DroppingInsertionPoint` component.